x/gamm/keeper/msg_server_test.go

```diff
@@ -305,7 +316,7 @@ func (suite *KeeperTestSuite) TestExitPool_Events() {
 			poolId:        1,
 			shareInAmount: sdk.NewInt(shareIn),
 			tokenOutMins:  sdk.NewCoins(sdk.NewCoin("foo", sdk.NewInt(tokenOutMinAmount))),
-			expectError:   true,
+			expectError:   types.ErrLimitMinAmount,
```

Comment:
Isn't this error returned wrapped with the sdk error? e.g `sdkerrors.Wrapf(types.ErrLimitMaxAmount, sharesLargerThanMaxErrFormat, sharesIn.Int64(), shareInMaxAmount.Uint64())` ?

Reply:
ah missed, updated